lib/webpacker/manifest.rb

```diff
@@ -13,32 +13,87 @@ def file_path
       Webpacker::Configuration.manifest_path
     end
 
-    def lookup(name)
+    # Converts the "name" (aka the pack or bundle name) to to the full path for use in a browser.
+    def asset_source(name)
+      output_path_or_url = Webpacker::Configuration.output_path_or_url
+      mapped_name = lookup(name)
+      "#{output_path_or_url}/#{mapped_name}"
```

Comment:
@gauravtiwari are you sure that the `mapped_name` does not include the subdirectory?